chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -993,3 +1003,18 @@ func (b *BitcoindNotifier) GetBlock(hash *chainhash.Hash) (*wire.MsgBlock,
 
 	return b.blockCache.GetBlock(hash, b.chainConn.GetBlock)
 }
+
+// SubscribeMempoolSpent allows the caller to register a subscription to watch
+// for a spend of an outpoint in the mempool.The event will be dispatched once
+// the outpoint is spent in the mempool.
+//
+// NOTE: part of the MempoolWatcher interface.
+func (b *BitcoindNotifier) SubscribeMempoolSpent(
+	outpoint wire.OutPoint) (*chainntnfs.MempoolSpendEvent, error) {
+
+	event := b.memNotifier.SubscribeInput(outpoint)
```

Comment:
What about unsubscribing?

Reply:
Yeah previously approach didn't work😂 now added unsubscribing methods and unsubscribed when the tx is confirmed, or the caller decide to no longer receiving updates.